blocks/core/ff_container/ff_container-dialog/ff_container-dialog.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var React = require('react');
+
+function generateClass(base, props){
+    var classNames = [];
+        classNames.push(base);
+        if (!!props.modifier) classNames.push(base + '--' + props.modifier);
```

Comment:
I can't remember the exact case, but there was an instance where the distinction between truthy and actual boolean value was required, so this is just to make that explicit.